src/SpacetimeDBClient.cs

```diff
@@ -65,6 +66,12 @@ public DbConnectionBuilder<DbConnection, Reducer> WithCompression(Compression co
             return this;
         }
 
+        public DbConnectionBuilder<DbConnection, Reducer> WithLightMode(bool light)
```

Comment:
I question why this needs to take a bool, tbh, since `false` is the default and so the only reason to call this is to assign `true`, but I also don't care.